broken/kokkos3-cuda.txt

```diff
@@ -0,0 +1,8 @@
+linux-64/kokkos-3.7.02-h1e7fabd_0.conda
+linux-64/kokkos-3.7.02-h22b3db6_1.conda
+linux-64/kokkos-3.7.02-h66b1f04_0.conda
+linux-64/kokkos-3.7.02-h69fb06d_1.conda
+linux-aarch64/kokkos-3.7.02-h5405a99_0.conda
+linux-aarch64/kokkos-3.7.02-h69b1e34_1.conda
+linux-ppc64le/kokkos-3.7.02-hc023538_1.conda
+linux-ppc64le/kokkos-3.7.02-hd810a49_0.conda
```

Comment:
Do both `build/numbers` need to be marked broken or only the `0` ones?

```suggestion
linux-64/kokkos-3.7.02-h1e7fabd_0.conda
linux-64/kokkos-3.7.02-h66b1f04_0.conda
linux-aarch64/kokkos-3.7.02-h5405a99_0.conda
linux-ppc64le/kokkos-3.7.02-hd810a49_0.conda
```

Reply:
Yes. All build numbers for this version are affected.

Reply:
Oh ok. Thought for some reason PR ( https://github.com/conda-forge/kokkos-feedstock/pull/24 ) was fixing this issue. Sounds like I misunderstood